cyw43/src/bus.rs

```diff
@@ -196,28 +244,45 @@ where
     }
 
     async fn backplane_readn(&mut self, addr: u32, len: u32) -> u32 {
+        defmt::debug!("backplane_readn addr = {:08x} len = {}", addr, len);
+
         self.backplane_set_window(addr).await;
 
         let mut bus_addr = addr & BACKPLANE_ADDRESS_MASK;
         if len == 4 {
-            bus_addr |= BACKPLANE_ADDRESS_32BIT_FLAG
+            bus_addr |= BACKPLANE_ADDRESS_32BIT_FLAG;
         }
-        self.readn(FUNC_BACKPLANE, bus_addr, len).await
+        
+        let val = self.readn(FUNC_BACKPLANE, bus_addr, len).await;
+
+        defmt::debug!("backplane_readn addr = {:08x} len = {} val = {:08x}", addr, len, val);
+
+        self.backplane_set_window(0x18000000).await; // CHIPCOMMON_BASE_ADDRESS
```

Comment:
the C code codes this, i guess we don't really need to? i can get rid of it and any of the other log lines if you want

Reply:
I don't think this set_window is needed, no.